context-free.md

```diff
@@ -1,22 +1,33 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+People who want to create primarily math/geometry-based images in a
+ (relatively) simple way. 
 
 ## What is easy to do in this language? Why is it easy?
-
+Creating images made up of a repitition of the same shape at a smaller
+size because the recursion available leads naturally to this.
 
 ## What is hard to do in this language? Why is it hard?
+Making images appear where you want them to is difficult because you
+either have to do a lot of math or a lot of experimentation. It is 
+also difficult to create a complex shape since the language only provides
+basic ones and as mentioned positioning is not intuitive. 
 
```

Comment:
I definitely agree with you. In fact, my image was an attempt to just place shapes where I wanted them and make a "plain" drawing which was definitely difficult due to the reasons you bring up.